genie-web/src/main/java/com/netflix/genie/web/agent/launchers/impl/LocalAgentLauncherImpl.java

```diff
@@ -173,6 +189,39 @@ public void launchAgent(@Valid final ResolvedJob resolvedJob) throws AgentLaunch
         }
     }
 
+    /**
```

Comment:
Since it's all data methods, why merging the two in here, as opposed to having a standalone indicator?